tests/functional/u/used/used_before_assignment.py

```diff
@@ -13,3 +13,8 @@ def inner():
         pass
 
 outer()
+
```

Comment:
# Importing the uuid module is slow, so import it only on request.
UUID: Callable[..., "uuid.UUID"] = None  # type: ignore[assignment]

def psycopg_false_positive():
        global UUID
        if not UUID:
            from uuid import UUID